data/json/recipes/recipe_ammo.json

```diff
@@ -630,6 +781,13 @@
     {"id":"SAW_M","level":1,"amount":1}
   ], "components" : [
     [
+      [ "superglue", 10 ],
+      [ "bone_glue", 10 ],
+      [ "thread", 20, false ],
```

Comment:
Because when I mentioned this change to kevin he suggested that it would just be easier to cut the threads. So that was the basis for this.